asistente_ladm_col/gui/qgis_model_baker/dlg_import_data.py

```diff
@@ -378,9 +394,15 @@ def updated_configuration(self):
 
         configuration.epsg = DEFAULT_EPSG
         configuration.inheritance = DEFAULT_INHERITANCE
-        configuration.create_basket_col = False
-        configuration.create_import_tid = False
-        configuration.stroke_arcs = True
+        configuration.create_basket_col = CREATE_BASKET_COL
+        configuration.create_import_tid = CREATE_IMPORT_TID
+        configuration.stroke_arcs = STROKE_ARCS
+
+        # Set java path
+        settings = QSettings()
+        java_path = settings.value('QgisModelBaker/ili2db/JavaPath', '', str)
```

Comment:
Can we use `get_java_path_from_qgis_model_baker()`?

Reply:
Done. Thanks